### PR TITLE
Add basic-commands folder as markdown format

### PR DESCRIPTION
**Pull request by TheMeinerLP**
## Description
Some documentation for FAWE from the basic-commands folder
## Submitter Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] New public fields and methods are annotated with `@since TODO`.
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: fastasyncworldedit/api/api-usage.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+
+```kt
+repositories {
+    mavenCentral()
+    maven { url = uri("https://repo.papermc.io/repository/maven-public/") }
+}
+
+dependencies {
+    implementation(platform("com.intellectualsites.bom:bom-1.18.x:{bomVersion}"))
```
Comment (NotMyFault) [inline fastasyncworldedit/api/api-usage.md:27]: That does no longer work. Same like below.

**Review by NotMyFault — COMMENTED** [review]
We should delete docinfo-footer.md, Placeholder.md and revise README.md to provide a useful index. Currently, all links are broken.

The basic commands aren't sorted in a folder, rather than the api page.

**Comment by NotMyFault** [comment]
Hey @TheMeinerLP,

any chance you can take a look at my feedback?

**Comment by TheMeinerLP** [comment]
Yeah ofc

**Comment by TheMeinerLP** [comment]
> Hey @TheMeinerLP,
> 
> any chance you can take a look at my feedback?

Hope this is now okay and useful for you

**Review by NotMyFault — APPROVED** [review]
Thanks, that's a great start, more pages to come later

**Pull request merged**